### ISSUE TITLE
route-tabset selectTab function 

### ISSUE THREAD

**Issue by louisamoros** [akveo/nebular#62]
I try to use the route-tabset component: https://github.com/akveo/nebular/blob/master/src/framework/theme/components/route-tabset/route-tabset.component.ts

but I think is not working as it should. I understand the purpose of the `selectTab(tab)` function but it does not act like it should.

```
selectTab(tab: any) {
  this.changeTab.emit(tab);
  // this line does not work and overrides the behaviour of routerLink in the template
  this.router.navigate([tab.route]);
}
```
Also I think it could be better to have this:

```
<ul>
  <li *ngFor="let tab of tabs"
      (click)="$event.preventDefault(); selectTab(tab)"
      routerLink="{{tab.route}}"
      routerLinkActive="active"
      [routerLinkActiveOptions]="{ exact: true }">
    <a href>{{tab.title}}</a>
  </li>
</ul>
```
instead of:
```
<ul>
  <li *ngFor="let tab of tabs"
      (click)="selectTab(tab)">
    <a [routerLink]="tab.route"
         routerLinkActive="active"
         [routerLinkActiveOptions]="{ exact: true }">{{tab.title}}</a>
  </li>
</ul>
```
 - No more `event.preventDefault`
 - Routing attribute are in the right semantic tag `<a>`

**Comment by louisamoros** [comment]
Hi @nnixaa, it's never too late. I will make one asap.

**Comment by nnixaa** [comment]
@louisamoros cool! Looking forward to it.

**Comment by nnixaa** [comment]
Fixed in 4.0
